testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -115,6 +116,7 @@ public abstract class AbstractGradleExecuter implements GradleExecuter, Resettab
         .build();
 
     private static final JvmVersionDetector JVM_VERSION_DETECTOR = GLOBAL_SERVICES.get(JvmVersionDetector.class);
+    private static final ObjectFactory objectFactory = GLOBAL_SERVICES.get(ObjectFactory.class);
```

Comment:
Isn't this an unused field now? Can we clarify with a comment?

Reply:
This was a leftover, I removed it.